securedrop/Dockerfile

```diff
@@ -20,6 +21,27 @@ RUN curl -LO https://launchpad.net/~ubuntu-mozilla-security/+archive/ubuntu/ppa/
     dpkg -i firefox*deb && apt-get install -f && \
     paxctl -cm /usr/lib/firefox/firefox
 
+
+COPY ./tor_project_public.pub /opt/
```

Comment:
Where is this file? Pulling down the branch locally, I can't find it anywhere. Was it omitted from an add? Recall that we already have the tor signing key at `install_files/ansible-base/roles/tor-hidden-services/files/tor-signing-key.pub`, although we'll have to consider the build context for the image and whether that's accessible here.

Reply:
Uh, yes, I added that in the `tbb_in_dev` branch (after I figured that I am missing that) :(